udp_driver/CMakeLists.txt

```diff
@@ -42,11 +50,16 @@ include_directories(include)
 ament_auto_add_library(${PROJECT_NAME} SHARED
   src/udp_socket.cpp
   src/udp_driver.cpp
-  examples/udp_driver_node.cpp)
+  src/udp_driver_node.cpp)
 ament_target_dependencies(${PROJECT_NAME} "Boost")
 
 target_compile_options(${PROJECT_NAME} PUBLIC "-O0" PRIVATE -Wno-sign-conversion -Wno-conversion)
 
+#rclcpp_components_register_node(udp_driver_node
```

Comment:
assuming you are planning on "compontent-izing" this later?

Reply:
Yes. Sorry this got mixed in. Planning on creating two nodes: sender and receiver. They will both be components and managed lifecycle nodes.